### PR TITLE
Adding meta title

### PR DESCRIPTION
**Pull request by decima**
still needs stylish but at least we have the component

### CODE REVIEW

File: assets/js/pages/article/read/ReadArticle.vue [hunk 1]
```diff
@@ -45,17 +45,23 @@
...
         mixins: [changePageTitleMixin],
         data() {
             return {
                 notFound: false
             }
         },
         computed: {
+            currentArticleMetadata() {
+                return {
+                    title: this.$store.getters.getCurrentArticle?.file?.metadata?.title
```
Comment (AlbanCrepel) [inline assets/js/pages/article/read/ReadArticle.vue]: Why returning an object ? Maybe we should just return this.$store.getters.getCurrentArticle?.file?.metadata here

File: assets/js/pages/article/read/ReadArticle.vue [hunk 2]
```diff
@@ -1,10 +1,10 @@
 <template>
     <div>
+        <article-title>{{ currentArticleMetadata.title }}</article-title>
```
Comment (AlbanCrepel) [inline assets/js/pages/article/read/ReadArticle.vue]: Why not putting that just before `<article-content/>` ? It seems more logical. And maybe we should display that only if there actually is a title in the metadata

File: assets/js/pages/article/read/ReadArticle.vue (hunk 1 above)
Comment (decima) [reply assets/js/pages/article/read/ReadArticle.vue]: The idea is to prepare multiple fields, that's why the currentArticleMetadata is not named currentArticleTitle :-)

What do you think about this?

File: assets/js/pages/article/read/ReadArticle.vue [hunk 3]
```diff
@@ -4,10 +4,12 @@
              v-if="$store.getters.getCurrentArticle"
              :key="$store.getters.getCurrentArticle.file.path"
              v-hotkey="keymap">
-
             <a-row type="flex">
                 <a-col class="article-content-col"
                        :style="{ width : $store.getters.getTocCollapsed ? '100%' : 'calc(100% - 250px)' }">
+                    <article-title v-if="$store.getters.getCurrentArticle.file.metadata.title">
+                        {{ $store.getters.getCurrentArticle.file.metadata.title }}
+                    </article-title>
```
Comment (AlbanCrepel) [inline assets/js/pages/article/read/ReadArticle.vue]: It seems that we use the ArticleTitle component only here..., why creating this component if it's just one `h1` html tag ? If we use it somewhere else, it's alright... But still, why passing a props then ? Couldn't we just display the value inside the component if the value displayed will always be equal to `this.$store.getters.getCurrentArticle.file.metadata.title` ?

**Pull request merged**